packages/edit-post/src/components/keyboard-shortcut-help-modal/style.scss

```diff
@@ -1,9 +1,12 @@
 .edit-post-keyboard-shortcut-help {
-
 	&__section {
 		margin: 0 0 2rem 0;
 	}
 
+	&__main-shortcuts .edit-post-keyboard-shortcut-help__shortcut-list {
+		// Push the shortcut to be flush with top modal header.
+		margin-top: -17px;
```

Comment:
Can we use `margin-top: -$panel-padding`which is -16px to avoid an hardcoded value?